app/screens/convert_gm_to_channel/convert_gm_to_channel_form/no_common_teams_form.tsx

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback} from 'react';
+import {useIntl} from 'react-intl';
+import {View, type ViewStyle} from 'react-native';
+
+import Button from '@app/components/button';
+import {useTheme} from '@app/context/theme';
+import {popTopScreen} from '@app/screens/navigation';
+
+import MessageBox from '../message_box/message_box';
```

Comment:
`message_box.tsx` can be renamed to `index.tsx` and this ends up being

```suggestion
import MessageBox from '../message_box';
```